patches/api/0054-Fix-upstream-javadocs.patch

```diff
@@ -46,6 +117,43 @@ index f84c37ad591f4d0e4062889941791a3aeb7c5be5..d9566b18e6109db824cbc1732666771b
       *
       * @return an array containing all previous players
       */
+diff --git a/src/main/java/org/bukkit/World.java b/src/main/java/org/bukkit/World.java
+index e6c9942f7820f2b8750c1bb0825c8bdbc6f4b99e..4ff3fce2afbac941ade5bda0882b78fcb4f078f2 100644
+--- a/src/main/java/org/bukkit/World.java
++++ b/src/main/java/org/bukkit/World.java
+@@ -1241,14 +1241,15 @@ public interface World extends RegionAccessor, WorldInfo, PluginMessageRecipient
+ 
+     /**
+      * Spawn a {@link FallingBlock} entity at the given {@link Location} of
+-     * the specified {@link Material}. The material dictates what is falling.
++     * the specified {@link MaterialData}. The material data dictates what is falling.
+      * When the FallingBlock hits the ground, it will place that block.
+      * <p>
+-     * The Material must be a block type, check with {@link Material#isBlock()
+-     * material.isBlock()}. The Material may not be air.
++     * The Material of the material data ({@link MaterialData#getItemType() data.getItemType()})
++     * must be a block type, check with {@link Material#isBlock() material.isBlock()}.
++     * The Material may not be air.
+      *
+      * @param location The {@link Location} to spawn the FallingBlock
+-     * @param data The block data
++     * @param data The material data
```

Comment:
nah, i try change this in upstream https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/pull-requests/670/overview but that PR stuck in.. not sure in what...

Reply:
i don't have access to those PRs, but if you commit a fix upstream before this one i will just rebase so not a problem. But since paper use a patch system everywhere even in API it's more a diff problem here than in spigot

Reply:
that PR is from 2021 then... well i hope gets merger its only a docs change because this methods use a copy-paste...